examples_tests/37.OptiXTriangle/main.cpp

```diff
@@ -213,7 +213,6 @@ int main()
 	OptixPipelineLinkOptions pipeline_link_options = {};
 	pipeline_link_options.maxTraceDepth = 5;
 	pipeline_link_options.debugLevel = OPTIX_COMPILE_DEBUG_LEVEL_FULL;
-	pipeline_link_options.overrideUsesMotionBlur = false;
```

Comment:
what happened to `overrideUsesMotionBlur ` in the newer SDK?

Reply:
its gone